### ISSUE TITLE
Content changes for GFLB's PDOM (e.g. deviations from GFL's PDOM)

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#77]
PDOM implementation of static and dynamic descriptions has begun for GFLB based on work completed for GFL. In an email we discussed some needed changes that are in progress. Here's a list of things I noticed on master November 2, 2019.

**Edited screen summary from the GFLB design document:**
> Gravity Force Lab: Basics is an interactive sim. It changes as you play with it. It has a Play Area and a Control Area. The Play Area has two spheres, a blue sphere labelled Mass 1 and a red sphere labelled Mass 2, and each sphere is held in place by a robot. Arrows representing force starts at center of each sphere and point directly at opposite sphere. Spheres can be moved closer or further from each other, and mass of each sphere can be increased or decreased. In Control Area there are checkboxes and buttons to show force values, show distance between spheres, set spheres to a constant size, and to reset sim. 

- [x] names of masses should be: mass 1 blue sphere and mass 2 red sphere
- [x] remove the sentence about the 10-meter ruler
- [x] change the sentence describing the force vectors 
- [x] change last sentence about Control Area

**Values for force, positions, and mass**
- [x] force values in PDOM need to match visual values. Force values are in Newtons andare rounded to a single decimal point
- [x] an exact position value is "kilometer mark" rather than a meter mark
- [x] mass values are in billions of kilograms rather than kilograms. e.g. 2 billion kilograms

**Comment by terracoda** [comment]
@mbarlow12, check items off as you go, and I'll verify when you are ready.

**Comment by terracoda** [comment]
@mbarlow12, thanks for getting this updated. Sounds really good. I'll go ahead close this issue.
